.pre-commit-config.yaml

```diff
@@ -5,13 +5,13 @@ repos:
       - id: isort
         args: [ "." ]
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.961
+    rev: v1.4.1
```

Comment:
would it be possible to just use the most recent snapshot there instead of a fixed revision?

Reply:
Initially wondered if the rev could be removed. Let me research further later.

I saw the rev was optional in a stack overflow but the official doc didn't address that.

Reply:
According to GPT, we can use: `rev: latest`
didn't tried it yet though :)

Reply:
it seems like `pre-commit autoupdate` is the only thing we could do to update versions, so maybe the best way is run `pre-commit autoupdate` via CI task and commit it on it...?

Anyway, this pr has nothing we can do more